amount.go

```diff
@@ -0,0 +1,7 @@
+package stripe
+
+// Amount describes a monetary amount in a specific currency in minor units.
+type Amount struct {
+	Currency Currency `json:"currency"`
+	Value    int64    `json:"value"`
+}
```

Comment:
This is part of the breaking change right? Can we add a changelog entry in the PR description?